src/plugins/data/public/index.ts

```diff
@@ -212,6 +212,12 @@ export {
   FieldFormat,
 } from '../common';
 
+/**
+ * Exporters (CSV)
+ */
+
+export * from './exports';
```

Comment:
Same comment re: using named exports from top-level `public/index.ts` and `server/index.ts`